endpoints.go

```diff
@@ -40,6 +40,10 @@ func endpoints(gcfg globalConfig) ([]string, error) {
 	return endpointsFromCluster(gcfg)
 }
 
+func isLocalEp(ep string) bool {
+	return strings.Contains(ep, "127.0.0.1") || strings.Contains(ep, "localhost")
+}
```

Comment:
Please also add an unit test.
```suggestion
func isLoopback(address string) bool {
	ip := net.ParseIP(address)
	if ip == nil {
		return false
	}
	
	return ip.IsLoopback()
}
```

Reply:
> Please also add an unit test.

Please let the unit test case to test `endpointsFromCluster`, thanks

Reply:
You marked this comment as resolved but It seems that you haven't resolved this comment?

Reply:
Hey, did you intentionally ignore this comment?